src/xopen/__init__.py

```diff
@@ -13,6 +13,7 @@
 import stat
 import signal
 import pathlib
+import subprocess
```

Comment:
Just noting, no change requested: Mixing `import subprocess` and `from subprocess import ...` is a bit inconsistent, but I can see why you wouldn’t want to import a function just named `run` at module level.